concourse/setup.py

```diff
@@ -452,6 +452,38 @@ def destroy_concourse_landscape(config_name: str, release_name: str):
     namespace_helper.delete_namespace(namespace=release_name)
 
 
+def deploy_webhook_dispatcher_landscape(
+    webhook_dispatcher_cfg: WebhookDispatcher,
+    chart_dir: str,
+    deployment_name: str,
+):
+    chart_dir = os.path.abspath(chart_dir)
```

Comment:
does it need to exist?

Reply:
yes it must exists before

Reply:
well.. how about util.existing_dir, then?